CSS/styles.css

```diff
@@ -0,0 +1,123 @@
+nav, h1 {
+
+  text-align: center;
+
+}
+
+h1 {
+
+  font-family: cursive;
+  font-size: 40px;
+
+}
+
+body {
+
+  background-color: tan;
+
+}
+
+hr {
+
+  color: gold;
+  border-width: 3px;
+  margin-bottom: 35px;
+  margin-top: 30px;
+
+}
+
+nav {
+
+  font-size: 25px;
+  font-family: garamond;
+
+}
+
+h2.indexh2 {
+
+  text-align: center;
+  font-family: garamond;
+  text-transform: uppercase;
+
+}
+
+#biography{
+
+  font-style: italic;
+  font-weight: bold;
+  text-decoration: none;
+
+}
+
+p {
+
+  font-family: garamond;
+  font-size: 20px;
+  margin-left: 30px;
+  margin-right: 30px;
+
+}
+
+h2.accomplishments {
+
+  font-family: garamond;
+  font-size: 30px;
+  margin-left: 20px;
+
+}
+
+ul {
+
+  font-family: garamond;
+  font-size: 20px;
+  margin-left: 20px;
+
+}
+
+.elizabeth {
+
+  height: 600px;
+  width: 500px;
+  margin-left: 440px;
+  border: solid gold 4px;
+
+}
+
```

Comment:
A preferred strategy here would be to center the element (You are missing your quotes around the attribute value.) as opposed to using a fixed margin on one side.  It may line up as the design dictates on some screens, but it will not on the rest.  🔨